tests/rptest/tests/datalake/disk_budget_test.py

```diff
@@ -107,7 +107,7 @@ def produce_until_staging_size(self, target_size):
         return current_size
 
     @cluster(num_nodes=2)
-    #@skip_debug_mode
+    @skip_debug_mode
```

Comment:
Squash?

Reply:
Also was changing the test configs intentional?

Reply:
fixed. changes were intentional, but only changing the time slice seemed to be important (to help avoid oom triggered translator finishing). i just change 60gb to 100gb too because 100gb seems "big", the goal of the value, but 60gb seems really specific.